bitnami/mongodb-sharded/templates/config-server/config-server-statefulset.yaml

```diff
@@ -114,6 +114,8 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: MONGODB_PORT_NUMBER
+              value: {{ $.Values.common.containerPorts.mongo | quote }}
```

Comment:
Why is the `$` needed here? It seems the context is the proper one

Reply:
Good catch, copy-paste